### PR TITLE
zh-trans: update content/zh/docs/admin/authorization/abac.md

### PR DESCRIPTION
**Pull request by SataQiu**
/cc @xichengliudui 
/assign @xichengliudui

### CODE REVIEW

File: content/zh/docs/admin/authorization/abac.md [hunk 1]
```diff
@@ -130,13 +130,13 @@ system:serviceaccount:<namespace>:<serviceaccountname>
...
 
-例如，如果要将 API 的 kube-system 完整权限中的默认服务帐户授予，则可以将此行添加到策略文件中:
+例如，如果要将 API 的 kube-system 完整权限中的默认服务帐户授予，则可以将此行添加到策略文件中 :
 
 ```json
 {"apiVersion":"abac.authorization.kubernetes.io/v1beta1","kind":"Policy","spec":{"user":"system:serviceaccount:kube-system:default","namespace":"*","resource":"*","apiGroup":"*"}}
 ```
 
-需要重新启动 apiserver 以获取新的策略行.
+需要重新启动 apiserver 以获取新的策略行 .
```
Comment (xichengliudui) [inline content/zh/docs/admin/authorization/abac.md]:
```suggestion
需要重新启动 apiserver 以获取新的策略行。
```

File: content/zh/docs/admin/authorization/abac.md [hunk 2]
```diff
@@ -20,95 +20,95 @@ content_template: templates/concept
...
     {"apiVersion": "abac.authorization.kubernetes.io/v1beta1", "kind": "Policy", "spec": {"user": "kubelet", "namespace": "*", "resource": "pods", "readonly": true}}
     ```
-3. Kubelet 可以读写事件:
+3. Kubelet 可以读写事件 :
 
     ```json
     {"apiVersion": "abac.authorization.kubernetes.io/v1beta1", "kind": "Policy", "spec": {"user": "kubelet", "namespace": "*", "resource": "events"}}
     ```
-4. Bob 可以在命名空间“projectCaribou"中读取 pod:
+4. Bob 可以在命名空间“ projectCaribou" 中读取 pod:
```
Comment (xichengliudui) [inline content/zh/docs/admin/authorization/abac.md]: “ projectCaribou"

File: content/zh/docs/admin/authorization/abac.md [hunk 3]
```diff
@@ -20,95 +20,95 @@ content_template: templates/concept
...
-Kubectl 使用 api-server 的 `/api` 和 `/apis` 端点进行协商客户端/服务器版本。 通过创建/更新来验证发送到API的对象操作，kubectl 查询某些 swagger 资源。 对于API版本"v1", 那就是`/swaggerapi/api/v1` ＆ `/swaggerapi/ experimental/v1`。
+Kubectl 使用 api-server 的 `/api` 和 `/apis` 端点进行协商客户端 / 服务器版本。 通过创建 / 更新来验证发送到 API 的对象操作，kubectl 查询某些 swagger 资源。 对于 API 版本 "v1", 那就是 `/swaggerapi/api/v1` ＆ `/swaggerapi/ experimental/v1`。
 
-当使用 ABAC 授权时，这些特殊资源必须明确通过策略中的 `nonResourcePath` 属性暴露出来(参见下面的[例子](#examples)):
+当使用 ABAC 授权时，这些特殊资源必须明确通过策略中的 `nonResourcePath` 属性暴露出来 ( 参见下面的[例子](#examples)):
 
-* `/api`，`/api/*`，`/apis`和`/apis/*` 用于 API 版本协商.
-* `/version` 通过 `kubectl version` 检索服务器版本.
-* `/swaggerapi/*` 用于创建/更新操作.
+* `/api`，`/api/*`，`/apis` 和 `/apis/*` 用于 API 版本协商 .
```
Comment (xichengliudui) [inline content/zh/docs/admin/authorization/abac.md]: 。(包括下面两行)

File: content/zh/docs/admin/authorization/abac.md [hunk 4]
```diff
@@ -20,95 +20,95 @@ content_template: templates/concept
...
 
-要允许任何经过身份验证的用户执行某些操作，请将策略组属性设置为 `"system:authenticated“`。
+要允许任何经过身份验证的用户执行某些操作，请将策略组属性设置为 `"system:authenticated “ `。
 
-要允许任何未经身份验证的用户执行某些操作，请将策略组属性设置为`"system:authentication“`。
+要允许任何未经身份验证的用户执行某些操作，请将策略组属性设置为 `"system:authentication “ `。
 
 要允许用户执行任何操作，请使用 apiGroup，命名空间，
-资源和 nonResourcePath 属性设置为 `“*"`的策略.
+资源和 nonResourcePath 属性设置为 ` “*"` 的策略 .
```
Comment (xichengliudui) [inline content/zh/docs/admin/authorization/abac.md]:
```suggestion
资源和 nonResourcePath 属性设置为 ` “*"` 的策略 。
```
 “*"，英文是这样的吗？

File: content/zh/docs/admin/authorization/abac.md [hunk 5]
```diff
@@ -20,95 +20,95 @@ content_template: templates/concept
...
 
 ## 授权算法
 
 请求具有与策略对象的属性对应的属性。
 
-当接收到请求时，确定属性。 未知属性设置为其类型的零值（例如: 空字符串，0，false）。
+当接收到请求时，确定属性。 未知属性设置为其类型的零值（例如 : 空字符串，0，false）。
 
-设置为`“*"`的属性将匹配相应属性的任何值。
+设置为 ` “*"` 的属性将匹配相应属性的任何值。
```
Comment (xichengliudui) [inline content/zh/docs/admin/authorization/abac.md]: “*"

File: content/zh/docs/admin/authorization/abac.md [hunk 6]
```diff
@@ -20,95 +20,95 @@ content_template: templates/concept
...
 
 检查属性的元组，以匹配策略文件中的每个策略。 如果至少有一行匹配请求属性，则请求被授权（但可能会在稍后验证失败）。
 
-要允许任何经过身份验证的用户执行某些操作，请将策略组属性设置为 `"system:authenticated“`。
+要允许任何经过身份验证的用户执行某些操作，请将策略组属性设置为 `"system:authenticated “ `。
```
Comment (xichengliudui) [inline content/zh/docs/admin/authorization/abac.md]: "system:authenticated “

File: content/zh/docs/admin/authorization/abac.md [hunk 7]
```diff
@@ -20,95 +20,95 @@ content_template: templates/concept
...
 
-要允许任何未经身份验证的用户执行某些操作，请将策略组属性设置为`"system:authentication“`。
+要允许任何未经身份验证的用户执行某些操作，请将策略组属性设置为 `"system:authentication “ `。
```
Comment (xichengliudui) [inline content/zh/docs/admin/authorization/abac.md]: "system:authentication “

**Comment by xichengliudui** [comment]
thanks.
/lgtm
/approve
/hold

**Comment by SataQiu** [comment]
DONE

**Comment by xichengliudui** [comment]
thanks
/lgtm
/hold cancel

**Pull request merged**